<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HFigure</class>
 <widget class="QWidget" name="HFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HFigure</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QCustomPlot" name="plot_lnglat" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QCustomPlot" name="plot_xy" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QCustomPlot" name="plot_course" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QCustomPlot" name="plot_course_rudder" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QCustomPlot" name="plot_velocity" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QCustomPlot" name="plot_ye" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
